lsp/src/main/java/org/aya/lsp/server/AyaServer.java

```diff
@@ -54,6 +54,8 @@
       workOps.setChangeNotifications(true);
       workCap.setWorkspaceFolders(workOps);
       cap.setWorkspace(workCap);
+      cap.setHoverProvider(true);
+      cap.setSignatureHelpProvider(new SignatureHelpOptions(Collections.singletonList(" ")));
```

Comment:
What's this list?

Reply:
Tell the frontend that we support hover and signature help (like Ctrl+P)